arquillian-environment-reporter/arquillian-environment-reporter-impl/src/test/java/org/arquillian/environment/reporter/impl/TestRunnerDetectorTest.java

```diff
@@ -0,0 +1,17 @@
+package org.arquillian.environment.reporter.impl;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestRunnerDetectorTest {
+
+    @Test
+    public void should_return_maven_as_runner() {
+
+        final TestRunnerDetector.TestRunner testRunner = TestRunnerDetector.detect();
+
+        assertThat(testRunner).isEqualTo(TestRunnerDetector.TestRunner.MAVEN);
```

Comment:
I understand from where it is coming - but if contributor, or any unaware person would run all tests from the IDE (like I just did) it will be nothing more than confusion and time wasted. That's why I am proposing to change `detect()` method. I understand we have to have an integration tests of some sort, b/c all the tools can change packaging anytime, but for unit tests it would be enough to just pass arbitrary stack as an argument.

For integration tests we can use JUnit assumptions maybe?